istioctl/cmd/proxyconfig.go

```diff
@@ -707,10 +707,18 @@ func bootstrapConfigCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			return configWriter.PrintBootstrapDump()
+			switch outputFormat {
+			case summaryOutput:
+				return configWriter.PrintProxyVersionInfo()
+			case jsonOutput:
+				return configWriter.PrintBootstrapDump()
+			default:
+				return fmt.Errorf("output format %q not supported", outputFormat)
+			}
 		},
 	}
 
+	bootstrapConfigCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", jsonOutput, "Output format: one of json|short")
```

Comment:
the default seems to be `json` here but when I run locally its outputting the summary... not sure why. But should continue to default to json